forms/jqm/element.js

```diff
@@ -19,6 +19,9 @@ define(function (require) {
   // view mixins
   var toggleClass = require('forms/mixins/view-helper-mixins').toggleClass;
 
+  var dirtyClassName = 'bm-formelement-dirty';
+  var pristineClassName = 'bm-formelement-pristine';
+  var invalidClassName = 'bm-formelement-invalid';
```

Comment:
The pre-ES2015 convention for constants is to name them like ALL_CAPS. Actually, even _with_ the ES2015 `const` keyword I think it's probably helpful to be able to visually identify constants like this.